### PR TITLE
Added content to the Benchmarking tutorial for different motion planners

### PR DESCRIPTION
**Pull request by raghavendersahdev**
This PR is a work done by Raghavender as a part of 2018 Google summer of code. As a part of this PR, there  are added sections for benchmarking different motion planners (CHOMP, STOMP, OMPL) available in MoveIt. 
This adds tutorial sections to benchmark CHOMP, STOMP and OMPL for a given benchmark (which includes common start, goal states, planning scenes, queries, etc.)

### CODE REVIEW

File: doc/benchmarking/benchmarking_tutorial.rst [hunk 1]
```diff
@@ -113,3 +117,50 @@ It is possible to customize a benchmark run by deriving a class from ``Benchmark
...
+
+  roslaunch panda_moveit_config demo.launch db:=true
+
+In the second terminal, run either of the two commands: ::
+
+  rosrun moveit_tutorials collision_scene_example.py sparse
+
+or: ::
+
+  rosrun moveit_tutorials collision_scene_example.py cluttered
```
Comment (davetcoleman) [inline doc/benchmarking/benchmarking_tutorial.rst]: great instructions!!

File: doc/benchmarking/benchmarking_tutorial.rst [hunk 2]
```diff
@@ -5,34 +5,38 @@ Benchmarking
 
 .. note:: To use this benchmarking method, you will need to download and install the ROS Warehouse plugin. Currently this is not available from Debians and requires a source install for at least some aspects. For source instructions, see `this page <http://moveit.ros.org/install/source/dependencies/>`_
 
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
+
 The `benchmarking package <https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_ros/benchmarks>`_ provides methods to benchmark motion planning algorithms and aggregate/plot statistics using the OMPL Planner Arena.
-The example below demonstrates how the benchmarking can be run for a Fanuc M-10iA.
+The example below demonstrates how the benchmarking can be run for a panda robot arm.
```
[the remark this answers is not in the document]
Comment (raghavendersahdev) [reply doc/benchmarking/benchmarking_tutorial.rst]: done

File: doc/benchmarking/benchmarking_tutorial.rst (hunk 1 above)
Comment (raghavendersahdev) [reply doc/benchmarking/benchmarking_tutorial.rst]: thanks

**Comment by mamoll** [comment]
Looks good to me.

**Pull request merged**